packages/chart/src/ChartUtils.ts

```diff
@@ -1877,12 +1855,67 @@ class ChartUtils {
     return axis;
   }
 
+  /**
+   * Creates a plotly layout template object based on a given theme.
+   * See https://plotly.com/javascript/reference/layout/#layout-template
+   * @param theme The theme to use for the layout template
+   */
+  makeDefaultTemplate(theme: Partial<ChartTheme>): Template {
+    /* eslint-disable camelcase */
+    const {
+      error_band_line_color,
+      ohlc_increasing,
+      ohlc_decreasing,
+      title_color,
+    } = theme;
+
+    return {
+      data: {
+        bar: [
+          {
+            marker: {
+              line: { color: 'transparent' },
+            },
+          },
+        ],
+        scatter: [
+          {
+            error_x: { color: error_band_line_color } as ErrorBar,
+            error_y: { color: error_band_line_color } as ErrorBar,
+          },
+        ],
+        ohlc: [
+          {
+            increasing: { line: { color: ohlc_increasing } },
+            decreasing: { line: { color: ohlc_decreasing } },
+          },
+        ],
+        pie: [
+          {
+            outsidetextfont: {
+              color: title_color,
+            },
+          },
+        ],
+        treemap: [
+          {
+            outsidetextfont: {
+              color: title_color,
+            },
+          },
+        ],
+      },
+      /* eslint-enable camelcase */
+      layout: this.makeDefaultLayout(theme),
+    };
+  }
+
   /**
    * Creates a plotly layout object based on a given theme.
    * See https://plotly.com/javascript/reference/layout/
    * @param theme The theme to use for the layout
    */
-  makeDefaultLayout(theme: ChartTheme): Partial<Layout> {
+  makeDefaultLayout(theme: Partial<ChartTheme>): Partial<Layout> {
```

Comment:
No need for partial on the param if the default template just expects the full theme